### PR TITLE
chore: update frontify eslint config to 0.17

### PR DESCRIPTION
**Pull request by SamuelAlev**

### CODE REVIEW

**Review by anxobotana — APPROVED** [review]
Code wise, files owned by `Guidelines Navigation & Themes` look good

File: packages/guideline-blocks-settings/src/helpers/customCoordinatesGetterFactory.ts [hunk 1]
```diff
@@ -13,21 +13,25 @@ export const customCoordinatesGetterFactory =
             const height = activeNode?.offsetHeight ?? 0;
 
             switch (event.code) {
```
Comment (ragi96) [inline packages/guideline-blocks-settings/src/helpers/customCoordinatesGetterFactory.ts:15]:
```suggestion
            switch (event.code as KeyboardCode) {
```

So we could remove the eslint-disable

**Review by ragi96 — APPROVED** [review]
G1 owned Files are looking good, just added 2 Nits which we also can tackle in the team

File: packages/app-bridge/src/AppBridgePlatformApp.ts [hunk 2]
```diff
@@ -126,18 +125,20 @@ export class AppBridgePlatformApp implements IAppBridgePlatformApp {
                 appBridgeVersion: 'v3',
             });
 
-            subscribe<InitializeEvent>(Topic.Init, PUBSUB_CHECKSUM).then(({ statePort, apiPort, context, state }) => {
-                this.apiMessageBus = new MessageBus(apiPort);
-                this.stateMessageBus = new MessageBus(statePort);
+            await subscribe<InitializeEvent>(Topic.Init, PUBSUB_CHECKSUM).then(
```
Comment (marguslant0) [inline packages/app-bridge/src/AppBridgePlatformApp.ts:128]: I noticed we're mixing `await` and `then` here. Is it intentional?

**Review by marguslant0 — APPROVED** [review]
💪

**Review by henser — APPROVED** [review]
I'm also happy for you 🙂

**Comment by SamuelAlev** [comment]
Small amount of conflicts, gonna do a new one

**Pull request closed without merging**